src/GroupAction/GroupActionCollection.php

```diff
@@ -167,7 +167,7 @@ public function submitted(NetteForm $form): void
 			return;
 		}
 
-		$values = (array) $form->getValues();
+		$values = $form->getUnsafeValues(null);
```

Comment:
This would mean you have to have nette forms 3.1.2 minimum, and I believe this won't be merged because of it. You should maybe check if getUnsafeValues exists and call it, if it doesn't then call getValues instead...